sr/comp/cli/show_league_table.py

```diff
@@ -0,0 +1,74 @@
+__description__ = "Show the current state of the league table"
+
+
+SORT_TYPES = {  # cli string: (sort column, sort direction)
+    'rank': (0, False),
+    'game': (2, True),
+    'team': (3, False),
+}
+
+
+def command(settings):
+    import os.path
+    from collections import Counter
+
+    from tabulate import tabulate
+
+    from sr.comp.comp import SRComp
+
+    comp = SRComp(os.path.realpath(settings.compstate))
+
+    league_table = []
+    for team in comp.teams.values():
+        scores = comp.scores.league.teams[team.tla]
+        league_pos = comp.scores.league.positions[team.tla]
+        league_table.append([
+            league_pos,
+            scores.league_points,
+            scores.game_points,
+            team.tla,
+            team.name,
+        ])
+
+    # sort rows
+    sort_col, sort_direction = SORT_TYPES[settings.sort]
+    league_table.sort(key=lambda row: row[sort_col], reverse=sort_direction)
+
+    # calculate ties
+    tie_count = Counter(rank for rank, _, _, _, _ in league_table)
+    ties = [rank for rank, count in tie_count.items() if count > 1]
+
+    # apply tie formatting
+    for team_data in league_table:
+        if team_data[0] in ties:
+            team_data[0] = f"={team_data[0]}"
```

Comment:
Can we fold this into the other formatting done below? (I think it would just be a case of adding `f"={rank}" if rank in ties else rank` over just `rank` in the constructed list? Might need to wrap that list at that point, but that's probably fine)

Alternatively, maybe it would make sense to have a formatting pass over all the data rather than that being spread between this loop and the comprehension below?